.github/workflows/reusable-mlir-tests.yml

```diff
@@ -103,9 +103,12 @@ jobs:
       - name: Build MLIR lit target
         run: cmake --build build --config ${{ matrix.coverage && 'Debug' || 'Release' }} --target mqt-core-mlir-lit-test-build-only
 
-      - name: Build MLIR unittests
+      - name: Build MLIR translation unittests
         run: cmake --build build --config ${{ matrix.coverage && 'Debug' || 'Release' }} --target mqt-core-mlir-translation-test
 
+      - name: Build MLIR wireiterator unittests
+        run: cmake --build build --config ${{ matrix.coverage && 'Debug' || 'Release' }} --target mqt-core-mlir-wireiterator-test
+
```

Comment:
This works but we should probably think about creating a single executable for MLIR unit tests.

Reply:
FYI: I've added a standalone `wireiterator` test here because it doesn't fit into the "translation" category.